src/components/list-group/README.md

```diff
@@ -138,7 +138,7 @@ means, such as additional text hidden using the `.sr-only` class.
 ## With badges
 
 Add [badges](/docs/components/badge) to any list group item to show unread counts, activity, and
-more with the help of some [utility classes](http://getbootstrap.com/docs/4.0/utilities/flex/).
+more with the help of some [utility classes](https://getbootstrap.com/docs/4.4/utilities/flex/).
```

Comment:
We may just want to point this to  `/docs/reference/utility-classes` so that we don't have to update this again :wink: 

Perhaps this:

```md
more with the help of some [flex utility classes](/docs/reference/utility-classes)
```

Reply:
All tweaked.